_includes/manuals/1.10/4.3.5.3_gsstsig.md

```diff
@@ -32,8 +33,21 @@ Once the keytab file has been created, test it using kinit:
 If this works, clear the Kerberos ticket cache once again using `kdestroy`.
 
 Store the keytab at `/etc/foreman-proxy/dns.keytab`, ensure permissions are 0600 and the owner is `foreman-proxy`.
+If you are using SELinux, do not forget to update the file context.
 
-The DNS zone `Dynamic Updates` option on the DNS zones can now be set to `Secure Only`. Now follow the steps below under `Proxy Configuration`.
+Now update the configuration in /etc/foreman-proxy/settings.d/dns.yml for the smart proxy:
+
+    :enabled: true
+    :dns_provider: nsupdate_gss
+    :dns_key: false
```

Comment:
These settings have changed in 1.10, they're now split into dns_nsupdate_gss.yml etc.